lbrynet/daemon/Publisher.py

```diff
@@ -54,6 +55,42 @@ def create_and_publish_stream(self, name, bid, claim_dict, file_path, claim_addr
     @defer.inlineCallbacks
     def publish_stream(self, name, bid, claim_dict, claim_address=None, change_address=None):
         """Make a claim without creating a lbry file"""
+        log.info('Starting publish for %s', name)
+
+        source = claim_dict['stream']['source'].get('source', None)
+        source_type = claim_dict['stream']['source'].get('sourceType', None)
```

Comment:
I think it's worth making a helper `ClaimDict.source_type` for this, similar to https://github.com/lbryio/lbryschema/blob/acb05b52f13bd0028c2b2d4d8939fb313c83c079/lbryschema/claim.py#L67